CHANGELOG.md

```diff
@@ -127,7 +127,8 @@
 
 ### Documentation
 
-- Fix broken links. @stevepiercy
+- Updated simple.md
+- fixes #3157 issue
```

Comment:
Never delete previous change log entries. Please restore it.

Please combine these lines into a single entry, and move it under the `Unreleased` main heading (because this change was not released yet, and it was definitely not released in 15.0.0-alpha.14 on 2022-03-10), creating a `Documentation` subheading if needed.

Reply:
Done! @stevepiercy